cmd/virt-handler/virt-handler.go

```diff
@@ -202,17 +201,6 @@ func (app *virtHandlerApp) Run() {
 	logger.V(1).Infof("hostname %s", app.HostOverride)
 	var err error
 
-	// Copy container-disk binary
-	targetFile := filepath.Join(app.VirtLibDir, "/init/usr/bin/container-disk")
```

Comment:
for reference https://github.com/kubevirt/kubevirt/pull/13626#pullrequestreview-2530921093